platform/windows/godot_res_template.rc

```diff
@@ -0,0 +1,28 @@
+#include "core/version.h"
+
+#ifndef RT_MANIFEST
+#define RT_MANIFEST 24
+#endif
+
+GODOT_ICON ICON platform/windows/godot.ico
+1 RT_MANIFEST "platform/windows/godot.manifest"
+
+1 VERSIONINFO
+FILEVERSION     GODOT_VERSION_MAJOR,GODOT_VERSION_MINOR,GODOT_VERSION_PATCH,0
+PRODUCTVERSION  GODOT_VERSION_MAJOR,GODOT_VERSION_MINOR,GODOT_VERSION_PATCH,0
+FILEOS          4
+FILETYPE        1
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "FileDescription",        GODOT_VERSION_NAME
+            VALUE "FileVersion",            GODOT_VERSION_NUMBER
```

Comment:
Maybe we should create `EngineDescription` and `EngineVersion` instead?
So users keep `File*` for their own project's metadata, but we keep a reference to Godot.

Likewise, would there be other values we want to keep with some `Engine` prefix?

Reply:
Though IIUC you said that when resource modification is enabled, we don't reuse any of this and write it manually, so if we want this we'd have to add it to the export code too I suppose.

Reply:
Non-standard fields will be invisible in UI. And yes, all are overwritten by exporter. So I left these only to make specific template version easy to identify.